PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx

```diff
@@ -366,86 +398,116 @@ void AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::DoJetLoop()
     TString groupname;
     AliJetContainer* jetCont = 0;
     TIter next(&fJetCollArray);
-
-	TClonesArray *triggerpatches = dynamic_cast<TClonesArray *>(InputEvent()->FindListObject("EmcalTriggers"));
-	if(!triggerpatches)
-		AliErrorStream() << "Trigger patch container EmcalTriggers not found in task " << GetName() << std::endl;
-
+    
+    TClonesArray *triggerpatches = dynamic_cast<TClonesArray *>(InputEvent()->FindListObject("EmcalTriggers"));
```

Comment:
Normally the AliAnalysisTaskEmcal takes care of reading objects from the event. Why is this needed?